src/types/widgets.ts

```diff
@@ -132,51 +132,51 @@ export interface Style {
   auto_refresh: string
   click_through_url: string
   enable_custom_tiles_per_page: boolean
-  load_more_type: string
+  load_more_type: "scroll" | "button" | "static"
```

Comment:
Add a new Type definition?

```
type LoadMoreType = "scroll" | "button" | "static"
```

and use it here

```
load_more_type: LoadMoreType
```

Reply:
Great Thanks